lib/systems/inspect.nix

```diff
@@ -13,8 +14,8 @@ rec {
     BigEndian    = { cpu = { significantByte = significantBytes.bigEndian; }; };
     LittleEndian = { cpu = { significantByte = significantBytes.littleEndian; }; };
 
-    Unix         = { kernel = { families = { inherit (kernelFamilies) unix; }; }; };
     BSD          = { kernel = { families = { inherit (kernelFamilies) bsd; }; }; };
+    Unix         = [ BSD Linux SunOS Hurd Cygwin ];
```

Comment:
I think it was the string ones that led me to avoid the inner `rec` to begin with, but that was probably an overkill move on my part :).